src/yomo.ts

```diff
@@ -0,0 +1,73 @@
+const completionSpec: Fig.Spec = {
+  name: "yomo",
+  description: "CLI interface for YoMo",
+  subcommands: [
+    {
+      name: "init",
+      description: "Initial an example StreamFunction",
+      args: {
+        name: "function name",
+        description: "StreamFunction name to initialize locally",
+      },
+      options: [
+        {
+          name: "--rx",
+          description: "Generate Rx code template",
+        },
+      ],
+    },
+    {
+      name: "build",
+      description: "Build a StreamFunction to WebAssembly",
+      args: {
+        name: ".go file",
+        description: "The .go file to build",
```

Comment:
See https://fig.io/docs/reference/generator/prebuilt-generators#figautocomplete-generators
```suggestion
        description: "The .go file to build",
        generators: filepaths({ extensions: ["go"] }),
```